src/standalone/Soil/Biogeochemistry/Biogeochemistry.jl

```diff
@@ -565,7 +567,7 @@ function ClimaLand.boundary_flux(
         Fields.field_values(Fields.level(Y.soilco2.C, p_len)),
         axes(Δz),
     )
-    C_bc = p.drivers.c_co2
+    C_bc = p.drivers.c_co2 # Kat, do we need to convert from kg C to mol CO2 here? 
```

Comment:
@kmdeck 
What do you think is the best approach?

Reply:
we can actually apply a preprocessing function to the driver as well... so each step it is read in and converted like so: https://github.com/CliMA/ClimaLand.jl/blob/394a295577c7fe4a998e6ed61949034074f8fa37/experiments/long_runs/soil.jl#L85

Otherwise we can specify that the c_co2 from the atmosphere must be in a certain set of units, and perform the conversion on this line?

Reply:
I did:
`C_bc = p.drivers.c_co2 * 83.26 # convert from kg C to mol CO2`